llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -2077,8 +2079,17 @@ void DwarfDebug::beginInstruction(const MachineInstr *MI) {
   unsigned LastAsmLine =
       Asm->OutStreamer->getContext().getCurrentDwarfLoc().getLine();
 
+  // Not-Key-Instructions functions inlined into Key Instructions functions
+  // should use default is_stmt handling. Key Instructions functions inlined
```

Comment:
I think the word "default" here is liable to become stale, is there another phrase we can use or invent? Keyless?